ingestion/src/metadata/ingestion/sink/elasticsearch.py

```diff
@@ -173,6 +177,15 @@ def __init__(
             ssl_context=ssl_context,
             ca_certs=self.config.ca_certs,
         )
+        if self.config.use_AWS_credentials:
+            credentials = boto3.Session().get_credentials()
+            http_auth = AWS4Auth(
+                region=self.config.region_name,
```

Comment:
I believe credentials in line 181 also has region configured by default. In case `region_name` is not present, you may want to use that as default.

Reply:
Ok, @abcabhishek I will add that.